cabal-install/tests/UnitTests/Distribution/Client/VCS.hs

```diff
@@ -693,3 +718,45 @@ vcsTestDriverDarcs mtimeChange verbosity vcs repoRoot =
                            }
     darcs = runProgramInvocation verbosity . darcsInvocation
 
+
+vcsTestDriverPijul :: Verbosity -> VCS ConfiguredProgram
```

Comment:
Nice. Having test is great. I'll take a look at CI so it won't fail

(Looks like there is no `apt-get install pijul` easy way of installing pijul on ubuntu)